nixos/tests/n8n.nix

```diff
@@ -1,13 +1,13 @@
 import ./make-test-python.nix ({ lib, ... }:
 
-with lib;
-
 let
   port = 5678;
 in
 {
   name = "n8n";
-  meta.maintainers = with maintainers; [ freezeboy k900 ];
+  meta.maintainers = builtins.attrValues {
+    inherit (lib.maintainers) freezeboy k900;
+  };
```

Comment:
IMO this code is now actually worse and not understandable for a beginner. I am against changing multi maintainer entries to use a function instead of being a list which uses with very locally.

Reply:
Beginners mostly copy and paste examples. Many of them still tops `with lib;` without knowing why - ans this is demonstrated by the use of `lib.mdDoc`.